test/testUtils.js

```diff
@@ -838,3 +838,66 @@ export const chartsDataFilterResponse = (overrides = {}) => ({
   lastModified: '',
   ...overrides,
 })
+
+export const createStudies = () => [
+  createStudy({
+    study: 'YA',
+    updatedAt: new Date('01-05-2024'),
+    participants: [
+      {
```

Comment:
can we use fixtures for these participant objects?

Reply:
[3fb9be8](https://github.com/AMP-SCZ/dpdash/pull/723/commits/3fb9be899eba72e300d84557422ec5fa4d84500d)